unit-test/dirSize.js

```diff
@@ -0,0 +1,16 @@
+import fastFolderSizeSync from 'fast-folder-size/sync.js'
+
+const buildDirs = ['android', 'chrome', 'chrome-mv3', 'firefox', 'integration', 'windows']
+const quotas = [512000, 680960, 512000, 512000, 512000, 512000]
```

Comment:
Could you add a comment to explain the idea of this and why you picked those values? I know we talked about 500kb in Asana, but to someone reading this code it might not be obvious what unit the numbers are in for example.